src/emptyClass.java

```diff
@@ -0,0 +1,2 @@
+public class emptyClass {
```

Comment:
Name of classes in Java start with capital letters. Please make this change. Thanks!

Reply:
Done... I think.